orchestra/static/orchestra/common/components/checklist/js/directives.js

```diff
@@ -21,7 +21,7 @@
           'comments': [{
             'author': 'jane',
             'timestamp': new Date(new Date() - 3 * 60 * 60 * 1000),
-            'text': "The hero image you've chosen doesn't fit " +
+            'text': 'The hero image you\'ve chosen doesn\'t fit ' +
```

Comment:
Our Python style guide suggests we do single quotes unless we'd have to escape, and then we do double quotes.  Any way to encode this in the `.jscsrc`?  If not, then such is life :)